apps/app/components/workspace/WorkspaceInvitationList.tsx

```diff
@@ -1,7 +1,16 @@
-import { Text, View } from "@serenity-tools/ui";
-import { FlatList, StyleSheet } from "react-native";
-import { TouchableOpacity } from "react-native-gesture-handler";
-import { WorkspaceInvitationListItem } from "./WorkspaceInvitationListItem";
+import {
+  IconButton,
+  List,
+  ListHeader,
+  ListItem,
+  ListText,
+  tw,
+  useIsDesktopDevice,
+  View,
+} from "@serenity-tools/ui";
+import { formatDistance, parseJSON } from "date-fns";
+import { isPast } from "date-fns/esm";
```

Comment:
@freuleinkorksi why not use `import { isPast } from 'date-fns'`?

Reply:
good catch! no reason, an auto-import fail